networkit/cpp/centrality/GroupHarmonicClosenessImpl.cpp

```diff
@@ -0,0 +1,596 @@
+/*
```

Comment:
This file implements a template specialization in a source file while the template is in a header file. This is an anti-pattern and has many pitfalls.

Move the implementation into the same file as the template declaration.